assets/javascripts/running.js

```diff
@@ -939,6 +946,12 @@ var messageToStatusVariable = [
     {
         msg: 'current_api_function',
         statusVar: 'currentApiFunction',
+        action: function (value, data) {
+            developerMode.currentApiFunctionArgs = '';
+            if ((value === 'assert_screen' || value === 'check_screen') && data.check_screen) {
+                developerMode.currentApiFunctionArgs = data.check_screen['mustmatch'];
```

Comment:
This is still using `data.check_screen['mustmatch']` instead of `data.check_screen.mustmatch`. By the way, this is mainly for readability and jshint (https://github.com/Martchus/openQA-helper#checking-for-javascript-errors).

Reply:
Ah, OK. I didn't ralize that I'm supposed to use the dot notation everywhere. Fixed.

There's no test for rendering currentApiFunction without arguments so I'll leave that to somebody who's more familiar with the testing framework.